core/src/ast_builder/expr/function.rs

```diff
@@ -110,6 +125,17 @@ mod tests {
         test_expr(actual, expected);
     }
 
+    #[test]
+    fn function_upper() {
+        // Upper
```

Comment:
Now it looks that this comment `// Upper` is unnecessary.
Everything else looks nice 👍 
Let's merge with this simple nit change.

Reply:
I totally agree with you, 
The comments are unnecessary at this point. 
I will fix them.